web/cypress/integration/route/online-debug.spec.js

```diff
@@ -42,6 +42,7 @@ context('Online debug', () => {
 
     // input uri with specified special characters
     urisWithSpecialChars.forEach((uri) => {
+      cy.get('#debugUri').clear();
```

Comment:
https://github.com/apache/apisix-dashboard/pull/1308/files#diff-f7a862c3853a71e325c09d7d25c05ccd96ba223ad611a71302d3547e8163275cR20-R28

Please take a look at this example, we may move all those mappers to a specified file in the future, but for now, we could use this way.

Reply:
It's a good idea to unified define `domSelectors`, fixed.